server/accounts.go

```diff
@@ -376,8 +376,9 @@ func (a *Account) AddServiceExport(subject string, accounts []*Account) error {
 	return a.AddServiceExportWithResponse(subject, Singleton, accounts)
 }
 
-// AddServiceExportWithresponse will configure the account with the defined export and response type.
-func (a *Account) AddServiceExportWithResponse(subject string, respType ServiceRespType, accounts []*Account) error {
+// AddServiceExportWithresponse will configure the account with the defined
+// export, response type, and latency info.
+func (a *Account) AddServiceExportWithResponseAndLatency(subject string, respType ServiceRespType, latency serviceLatency, accounts []*Account) error {
```

Comment:
There is a function to add on latency after you create the export. I would probably do that versus adding on to the above.